include/jemalloc/internal/arena_structs_b.h

```diff
@@ -123,6 +123,13 @@ struct arena_s {
 	 */
 	unsigned		nthreads[2];
 
+	/*
+	 * When percpu_arena is enabled, to amortize the cost of reading /
+	 * updating the current CPU id, track the most recent thread accessing
+	 * this arena, and only read CPU if there is a mismatch.
+	 */
+	tsdn_t		*last_thd;
```

Comment:
What happens if a thread is joined, a new one started, and we reuse the same tsd?  Is there still a fall back to call sched_getcpu every X accesses?